mittab/libs/outround_tab_logic/checks.py

```diff
@@ -0,0 +1,126 @@
+from mittab.apps.tab.models import *
+from mittab.libs.errors import PrevRoundNotEnteredError
+
+
+def lost_teams():
+    return [outround.loser.id
+            for outround in Outround.objects.all()
+            if outround.loser]
+
+
+def have_enough_judges_type(type_of_round):
+    lost_ids = [outround.loser.id
```

Comment:
```suggestion
    loser_ids = [outround.loser.id
```

Nit: imo this is a clearer name, when I read it at first I was thinking it was like the ID was missing from the db or something

Reply:
(make sure to get the other uses of this if you commit the change)